debian/rules

```diff
@@ -14,9 +14,11 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
-#	dotnet build --configuration $(CONFIG) $(CURDIR)/MediaBrowser.sln --output='$(CURDIR)/usr/lib/emby-server/bin'
-	dotnet publish --configuration $(CONFIG) $(CURDIR)/MediaBrowser.sln --output='$(CURDIR)/usr/lib/emby-server/bin'
+	git submodule init
+	git submodule update
+	dotnet build --configuration $(CONFIG) $(CURDIR)/MediaBrowser.sln --output='$(CURDIR)/usr/lib/jellyfin/bin'
```

Comment:
I'm not sure that `dotnet build` is needed, seems that `dotnet publish` might be enough

Reply:
OK I wasnt 100% sure about this, will just use `publish`.